modin/core/storage_formats/pandas/parsers.py

```diff
@@ -534,6 +534,13 @@ def _convert_cell(cls, cell, convert_float):
 
         return cell.value
 
+    @property
+    def need_rich_text_param(self):  # noqa: GL08
```

Comment:
What about a docsting for this method, where we would describe the need of checking openpyxl's version.

Reply:
added